deploy-config/deploy-config-holesky.toml

```diff
@@ -69,34 +70,22 @@ execution_delay = 900
 quorum = 1
 
 [[tiebreaker.committees]] # Influencers (participants from Devnet dry run multisig)
-members = [
-    "0x14df797E7910D320931016E87133D0977eCa3c54",
-    "0x6FD09F5b4341C79CBDf679bccC2394C6FD7D0016",
-    "0xe8B2097674daF8fC3DcCDDf05a2cBeF8253A4bAB",
-    "0x7Cd64b87251f793027590c34b206145c3aa362Ae",
-    "0x526d46eCa1d7969924e981ecDbcAa74e9f0EE566",
-    "0x6389CBcF38172a1e8eCF1C34c00cd7f603AfB80C",
-]
+members = ["0x61E3340206b32d293C9f1F8C4Ef76Aabc65A53cb"]
 quorum = 1
 
 [[tiebreaker.committees]] # Node Operators (participants from Devnet dry run multisig)
-members = [
-    "0x14df797E7910D320931016E87133D0977eCa3c54",
-    "0x6FD09F5b4341C79CBDf679bccC2394C6FD7D0016",
-    "0xe8B2097674daF8fC3DcCDDf05a2cBeF8253A4bAB",
-    "0x7Cd64b87251f793027590c34b206145c3aa362Ae",
-    "0x526d46eCa1d7969924e981ecDbcAa74e9f0EE566",
-    "0x6389CBcF38172a1e8eCF1C34c00cd7f603AfB80C",
-]
+members = ["0x61E3340206b32d293C9f1F8C4Ef76Aabc65A53cb"]
 quorum = 1
 
 [[tiebreaker.committees]] # Protocols (participants from Devnet dry run multisig)
-members = [
-    "0x14df797E7910D320931016E87133D0977eCa3c54",
-    "0x6FD09F5b4341C79CBDf679bccC2394C6FD7D0016",
-    "0xe8B2097674daF8fC3DcCDDf05a2cBeF8253A4bAB",
-    "0x7Cd64b87251f793027590c34b206145c3aa362Ae",
-    "0x526d46eCa1d7969924e981ecDbcAa74e9f0EE566",
-    "0x6389CBcF38172a1e8eCF1C34c00cd7f603AfB80C",
-]
+members = ["0x61E3340206b32d293C9f1F8C4Ef76Aabc65A53cb"]
```

Comment:
Consider using addresses of signers like on previous devnets instead of multisig here